docs/get-started/install.rst

```diff
@@ -66,8 +66,8 @@ to vectors, but it's not required to drive the plotter.
 Install the required system libraries (listed at https://www.piwheels.org/project/Pillow/), followed by
 Pillow itself::
 
-    sudo apt install libwebp6 libtiff5 libjbig0 liblcms2-2 libwebpmux3 libopenjp2-7 libzstd1 libwebpdemux2
-    pip install pillow
+    sudo apt install libwebp6 libtiff5 libjbig0 liblcms2-2 libwebpmux3 libopenjp2-7 libzstd1 libwebpdemux2 libjpeg-dev
```

Comment:
What system were you using, if `libjpeg-dev` were required?

Reply:
Linux Mint, I was adding deps one at a time trying to get pillow to build